public/docs/_examples/forms/dart/lib/hero.dart

```diff
@@ -0,0 +1,19 @@
+library hero_form.hero;
+
+class Hero {
+  num number;
+  String name;
+  String power;
+  String alterEgo;
+
+  Hero(this.number, this.name, this.power, {this.alterEgo});
+
+  Map toJson() {
```

Comment:
I'll add the arrow. It's not needed for the final code output, but in the TS example they add it for debugging, then remove towards the end. I wasn't how one to one the Dart docs are going to be, so I put it there just in case.